foundry.toml

```diff
@@ -26,6 +26,11 @@ ignored_warnings_from = ['contracts/lib']
 optimizer = true
 
 # See more config options https://github.com/foundry-rs/foundry/tree/master/config
+#
+[invariant]
+show_metrics = true
+fail_on_revert = true
```

Comment:
This is important. It means that if we have an uncaught error in the fuzz tests it would revert. I guess same caveats as with normal foundry testing reverts apply and it would not catch reverts from internal function calls.